apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,42 +1,333 @@
+                  value={field.value !== undefined ? field.value : selectedProvider.displayName}
+                  // error={showErrors && fieldState.error?.message}
+                  type="text"
+                  data-test-id="title"
+                  placeholder="Enter workflow name"
+                  // disabled={readonly}
+                />
+              );
+            }}
+          />
+          <ActionIcon
+            variant={'transparent'}
+            onClick={() => {
+              navigate('/integrations');
+            }}
+          >
+            <Close color={colors.B40} />
+          </ActionIcon>
+        </Group>
+        <Group mb={16} spacing={16}>
+          <IntegrationChannel name={CHANNEL_TYPE_TO_STRING[selectedProvider.channel]} type={selectedProvider.channel} />
+          <IntegrationEnvironmentPill
+            name={
+              environments?.find((environment) => environment._id === selectedProvider.environmentId)?.name ||
+              'Development'
+            }
+          />
+        </Group>
+        <CenterDiv>
+          <ActiveWrapper active={isActive}>
+            <Controller
+              control={control}
+              name="active"
+              render={({ field }) => <Switch checked={isActive} data-test-id="is_active_id" {...field} />}
+            />
+            <StyledText data-test-id="connect-integration-form-active-text">
+              {isActive ? 'Active' : 'Disabled'}
+            </StyledText>
+          </ActiveWrapper>
+          <Controller
+            control={control}
+            name="identifier"
+            render={({ field }) => (
+              <InputWrapper>
+                <Input
+                  label="Instance key"
+                  error={errors.identifier?.message}
+                  rightSection={
+                    <CopyWrapper onClick={() => identifierClipboard.copy(field.value)}>
+                      {identifierClipboard.copied ? <Check /> : <Copy />}
+                    </CopyWrapper>
+                  }
+                  {...field}
+                />
+              </InputWrapper>
+            )}
+          />
+          {selectedProvider?.credentials.map((credential: IConfigCredentials) => (
+            <InputWrapper key={credential.key}>
+              <Controller
+                name={`credentials.${credential.key}`}
+                control={control}
+                render={({ field }) => <IntegrationInput credential={credential} errors={errors} field={field} />}
+              />
+            </InputWrapper>
+          ))}
+        </CenterDiv>
+        <Group mt={16} position="apart">
+          <Text>
+            Explore our{' '}
+            <a
+              style={{
+                color: colors.error,
+              }}
+              href={selectedProvider.docReference}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              set-up guide
+            </a>
+          </Text>
```

Comment:
```suggestion
         <Center inline>
            <Text mr={5}>Explore our</Text>
            <Text gradient>
              <a
                style={{
                  color: colors.error,
                }}
                href={selectedProvider?.docReference}
                target="_blank"
                rel="noopener noreferrer"
              >
                set-up guide
              </a>
            </Text>
          </Center>
```

This should work. The Text is from the design system